features.md

```diff
@@ -8,11 +8,35 @@
 
 * ### Registering Patients
 
-   1. Entering Patient Details
+   You can register new patients by clicking on the **Register Patient** button in the home menu. When online, created patients are automatically synchronized to the server wih a unique patient ID. When offline, created patients are saved in the user's local database and are synced when the user is connected to the internet. See [Working Offline](#working-offline) for more details.
 
-   2. Adding Patient Photo
+   #### 1. Entering Patient Details
 
-   3. Finding Similar Patients
```

Comment:
Same as above

Reply:
Changed to a header [here](https://github.com/openmrs/openmrs-android-client-user-guide/pull/16/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R35)